File: doc/Examples_SPL/Expresiones/Expresiones/ShortCircuit/integerShortEval.cs
using System;
using Expresiones;


namespace Expresiones
{
    /**
     * Clase que define IntegerShortEval
     **/
    public class IntegerShortEval : IntegerEval, IBinaryOperatorEval
    {
        //int constante;
        /**
         * Constructor
         * */
        public IntegerShortEval (int constante)
            : base(constante)
        {
            //constan = constante;
            
        }
        /**
       * Método que retorna el valor del entero
       * */
        public override int eval()
        {
            return constante;
        }
    }
}

File: trunk/doc/Examples_SPL/expressionsCaesarJ/src/shortCircuit/IntegerShortEval.java
cclass shortCircuit.ShortCircuit;

public cclass IntegerShortEval extends IntegerEval & ExpressionShortEval {
	protected int constante;
	public IntegerShortEval(int constante){
		this.constante=constante;
	}
	public int eval(){
		return constante;
	}
}
